docs/Conventions.md

```diff
@@ -42,37 +43,41 @@ Array.filter([|1,2,3|], ~f=Int.isOdd)
 |> Array.map(~f=(number) => number * 3);
 ```
 
-## exn is for exception 
+## exn is for exception
 
 The type for exceptions in OCaml is actually called `exn`.
 
 You can read more about exceptions in [Real World OCaml](http://dev.realworldocaml.org/error-handling.html#scrollNav-2) or in the [OCaml manual](https://caml.inria.fr/pub/docs/manual-ocaml/coreexamples.html#s:exceptions).
 
 ## sep is for separator
 
-[`Array.intersperse`](/api#Array.intersperse), [`Array.join`](/api#Array.join) and their [`List`](/api#List) counterparts all take an element that will be used as a seperator. 
+[`Array.intersperse`](/api#Array.intersperse), [`Array.join`](/api#Array.join)
+and their [`List`](/api#List) counterparts all take an element that will be used
+as a separator.
 
 Since this is a bit of a mouthful for a pretty commonly used function it gets shortened to `sep`.
 
 ## Function suffixes
 
-Some functions come in multiple flavors. 
+Some functions come in multiple flavors.
 
 Some of these flavors are so common they are distinguished by a really short suffix.
 
-### ___2 is an alternative behaviour
+### \_\_\_2 is an alternative behaviour
```

Comment:
Is this some auto-formatting from an editor?

It seems to be causing the website-build to fail, could you revert the `\_`s? (Or figure out what needs to change on the website)

Reply:
Weird, I think this is due to a Markdown plugin I have in Neovim. I'll revert this chane.